### DISCUSSION TITLE
Any architecture that does Points, not boxes or segments.

### DISCUSSION THREAD

**Discussion by ExtReMLapin** [ultralytics/yolov5#10092] [Q&A]
Hello,
Is there any model/architecture (I will train it from scratch) that could just take points instead of boxes ?

Thanks

**Comment by ExtReMLapin** [comment]
My point is the following, i'm annotating small objects that could objectively annotated from 0.5 times the current size to 3.0 times the current size, I get better results (recall) when I increase all annotation sizes by a ratio of 3, but I find doing this is kind of dirty.
This is why i'm wondering if I can annotate points that will "somehow" decide itself which area to take around the point

I found something called `landmark annotations`, is it what i'm looking for ?

**Reply by glenn-jocher** [reply]
@ExtReMLapin you might just need to play around with NMS settings like IOU threshold and confidence etc to get the desired result. For pose there are some models like https://github.com/TexasInstruments/edgeai-yolov5 but not for generic point regression.

**Reply by ExtReMLapin** [reply]
Thank you glenn !
